R/analyze_compile_one.R

```diff
@@ -44,7 +49,7 @@ deepstate_fuzz_fun<-function(fun_path,seed=-1,time.limit.seconds=-1){
   }
   cat(sprintf("running the executable .. \n%s\n",run.executable))
   system(run.executable)
-  execution <- if(file.exists(test_harness_path)){
+  execution <- if(file.exists(test_harness.o) && file.exists(test_harness_path)){
     basename(fun_path)
   }else{
     "failed"
```

Comment:
why not stop() instead? when would this fail?
what if there is a function called "failed" ? 
if you really have to return a character string for failure, how about empty string "" ?

Reply:
This fails when the object file and test harness are not created: Sometimes executable and the object file will not be created if there are missing dependencies or headers. To make sure both of them exist and there is a successful execution.
 
As we loop through all the functions in the packages we need to keep track of what functions are executed and what is not.  

Yes, I shouldn't use failed it's true that a function name can be "failed". I'll return an empty string.

Reply:
or missing value

Reply:
Returning a missing value.